graylog2-server/src/main/java/org/graylog2/database/PaginatedList.java

```diff
@@ -80,13 +79,8 @@ public String toString() {
 
     @JsonAutoDetect
     public class PaginationInfo {
```

Comment:
Could this be converted to an Auto Value class in order to get rid of the custom `equals()`/`hashCode()`/ `toString()` implementations?

Reply:
Aye, will do.